plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/BrowseMarketplace.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Connect store to WooCommerce.com Task
```

Comment:
Shouldn't the doc explaining the "Browse the WooCommerce Marketplace" instead of "Connect store"?